storage/src/blockchain_db_adapter.cpp

```diff
@@ -188,8 +188,6 @@ char KeyManipulator::extractTypeFromKey(Key _key) {
 BlockId KeyManipulator::extractBlockIdFromKey(Key _key) {
   size_t offset = _key.length() - sizeof(BlockId);
   BlockId id = *(BlockId *)(_key.data() + offset);
-
-  LOG_DEBUG(logger_, "Got block ID " << id << " from key " << _key  << ", offset " << offset);
```

Comment:
I would not remove those logs, they help to debug. Don't we have an additional log level, lower than DEBUG? VERBOSE or TRACK?